src/uptane/tufrepository.cc

```diff
@@ -183,15 +183,6 @@ void TufRepository::initRoot(const Json::Value& new_content) {
                                                                << requiredThreshold);
       throw IllegalThreshold(name_, "root.json contains a role that requires too many signatures");
     }
-    for (Json::ValueIterator it_keyid = (*it)["keyids"].begin(); it_keyid != (*it)["keyids"].end(); ++it_keyid) {
```

Comment:
Ok, then this will significant some changes in the repo because uptane says a keyid is just the hash of the public key, tuf says it's the hash of the canonical json of the keys, this would be a new ticket.